components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -122,11 +122,11 @@ const PrepopulatedEngine qwant = {
 };
 
 const PrepopulatedEngine startpage = {
-    L"StartPage",
+    L"Startpage",
     L":sp",
     "https://www.startpage.com/favicon.ico",
     "https://www.startpage.com/do/"
-    "dsearch?query={searchTerms}&cat=web&pl=opensearch",
+    "search?q={query}&segment=startpage.brave",
```

Comment:
Sorry, didn't look before, but all other engines seem to use `{searchTerms}`, does it matter that we switch to `{query}` here?

Reply:
oh wow - yes, you're 100% right 🤦‍♂️ I copy pasted from an email (busted!) 😢 One moment here...

Reply:
OK fixed - thanks again 😄 This time I compiled + tested the implementation to verify it works as expected